Assets/Editor/BuildTiltBrush.cs

```diff
@@ -859,6 +857,11 @@ static void CommandLine()
             target = BuildTarget.StandaloneWindows64;
         }
 
+        if (target == BuildTarget.Android)
+        {
+            EditorUserBuildSettings.androidCreateSymbols = AndroidCreateSymbols.Debugging;
```

Comment:
I think we always want this set, even if it's not a debug build. It doesn't change anything about the executable files, just the (extra) symbols.zip

Reply:
This was moved from elsewhere in the file where it was previously only set for Oculus platforms, I think it's always generated regardless of debug or not!

Reply:
Right, and that's good! But we used to set it to true, and now it's only set if Debugging is enabled.

But there's a simple test; is it generated now? If so, let's merge. If not, let's go back to just "true".

I'll check tomorrow am, if you don't get to it first.

Reply:
the `true` method is deprecated in 2021, now you can choose either `Public` or `Debugging`, of which I assume Debugging is what we want! https://docs.unity3d.com/2021.2/Documentation/ScriptReference/AndroidCreateSymbols.html

Reply:
oh, ok, I totally misunderstood this! Never mind :-)